content/zh/activities/sofastack-open-source-7th-anniversary-upcoming-events/index.md

```
@@ -0,0 +1,88 @@
+
+📅 ​**时间**​：2025/4/26 15:00-18:00
+
+📍 **​地址：​**浙江省杭州市 · 云栖小镇国际会展中心 A 馆 1 F 慧云厅
+
+## 活动报名
+
+无论您是见证 SOFAStack 成长的『元老级』开发者，还是刚加入社区的『新鲜血液』，诚挚地邀请您加入 SOFAStack 七周年庆生活动，以热爱续写属于我们的精彩篇章！我们还为大家准备了 SOFAStack 七周年定制礼物🎁及茶歇，欢迎大家到现场一同交流分享👐👐
+
+![图4](https://img.alicdn.com/imgextra/i1/O1CN01N4yOCO22bGASmDwEs_!!6000000007138-2-tps-990-814.png)
+
+*（图片仅供参考，以收到实物为准）*
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)